cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -5085,6 +5085,14 @@
     </description>
   </property>
 
+  <property>
+    <name>feature.support.and.triggers.enabled</name>
```

Comment:
can you add more context to this feature flag name? maybe feature.pipelines.and.triggers.enabled ? I'll send you the naming guideline doc on chat

Reply:
Thanks! updated